src/test/resources/expected_helm_output/bitbucket/output.yaml

```diff
@@ -124,6 +128,10 @@ spec:
             - name: shared-home
               mountPath: "/var/atlassian/application-data/shared-home"
           env:
+            - name: KUBE_POD_NAME
```

Comment:
How this is utilised?

Reply:
This will define an environment variable named `KUBE_POD_NAME` in the container. This is done here to address the persistent identifier issue in ZDU. 
There is still a work to do that is  adding a jvm parameter (`-Dcluster.node.name=KUBE_POD_NAME`) in bitbucket startup script and involve in in node persistent identifier logic in ZDU. 
Hazelcast will assign pod name which is unique in each restart to node name in clusterInfo. Then what we have to do is involving node name into detect returning nodes in ZDU to handle the offline nodes display issue after a node is upgraded and is running. We have a ticket already to address this.